source/Pages/compute_on_spider.rst

```diff
@@ -57,8 +55,11 @@ SBATCH directive      Functionality         Usage example
 ``-p <partition>``    partition selection   ``#SBATCH -p infinite`` (the job will run max for 720 hours)
 ``-p <partition>``    partition selection   ``#SBATCH -p short`` (the job will run max for 12 hours)
 ``-p <partition>``    partition selection   ``#SBATCH -p interactive`` (the job will run max for 12 hours)
+``-p <partition>``    partition selection   ``#SBATCH -p gpu_v100`` (the job will run on V100 nodes with a max of 120 hours)
```

Comment:
Ok for now this makes sense, but if it does not exist already we should recommend users to use sinfo to check this information and provide sufficient documentation on how to interpret the sinfo output. I.e., I would not be surprised if we at some point would also need a gpu_a100_infinite partition and hence this info ages quickly